powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -393,6 +393,39 @@ void testIdentityVerificationCleanup() throws Exception {
         powerAuthClient.removeActivation(activationId, "test");
     }
 
+    @Test
+    void testIdentityVerificationMaxAttemptLimit() throws Exception {
```

Comment:
Maybe using `assertThrows` could help to validate more than just a thrown exception?

https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertThrows-java.lang.Class-org.junit.jupiter.api.function.Executable-